### ISSUE TITLE
--glob not processing absolute paths correctly

### ISSUE THREAD

**Issue by Olie440** [BurntSushi/ripgrep#479]
Glob appears to calculating all paths relatively even if the start with `/`:
![image](https://cloud.githubusercontent.com/assets/6021283/25894174/bec21a2e-3572-11e7-9ddd-639498782efc.png)

OS: MacOS 10.12.4 (darwin 16.5.0)
RipGrep version: ripgrep 0.5.1

**Comment by BurntSushi** [comment]
I don't think this is a bug. I don't see how to implement this behavior. Notice that `grep` has the same behavior:

```
$ tree
.
└── foo

0 directories, 1 file
$ cat foo
test
$ rg test -g '!/tmp/ripgrep-479/test'
foo
1:test
$ grep -r test --exclude /tmp/ripgrep-479/test
foo:test
```

Perhpas this is a documentation bug.
